File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    
    for _ in range(len(commands)):
        i = commands[_][0]
        j = commands[_][1]
        k = commands[_][2]
        
        temp = array[i-1:j]
        temp.sort()
        answer.append(temp[k-1])
    
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.cpp
#include <bits/stdc++.h>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    for (auto command : commands){
        vector<int> temp(array.begin() + command[0] - 1, array.begin() + command[1]);
        sort(temp.begin(), temp.end());
        answer.push_back(temp[command[2] - 1]);
    }
    return answer;
}